src/hooks/useSorting.ts

```diff
@@ -47,6 +48,10 @@ const columns: SortColumn[] = [
     sortKey: 'DATO',
     sortingText: 'Frist/Dato',
   },
+  {
+    sortKey: 'STATUS',
+    sortingText: 'Status',
```

Comment:
Usikker på hva som blir best her 🤔 Jeg tenker at den gjenspeiler hendelsen det gjelder, med litt ekstra informasjon
```suggestion
    sortingText: 'Hendelse',
```